### PR TITLE
[storage] [frontend] AIP Get/List/Create/Update/Delete rename

### PR DESCRIPTION
**Pull request by glimchb**
See #174

Resource-oriented design https://google.aip.dev/121

```
for word in Get List Create Update Delete; do
  for i in NVMeSubsystem NVMeController NVMeNamespace VirtioBlk VirtioScsiController VirtioScsiLun; do
    sed -i "s/${i}${word}/${word}${i}/g" v1alpha1/frontend_*.proto ;
  done;
done
```

Signed-off-by: Boris Glimcher <Boris.Glimcher@emc.com>

### CODE REVIEW

**Review by mestery — APPROVED** [review]
At this point, I'm just stamping this type of PR.

File: storage/Makefile [hunk 1]
```diff
@@ -15,6 +15,6 @@ all:
 	curl -kL https://github.com/googleapis/googleapis/archive/master.tar.gz | tar --strip=1 -zxvf - googleapis-master/google/api
 	mv google "${PWD}"/v1alpha1/
 	docker run --user=$$(id -u):$$(id -g) --rm --entrypoint=sh -v "${PWD}/../common/v1":/common -v "${PWD}"/v1alpha1/:/out -w /out ghcr.io/docker-multiarch/google-api-linter:1.36.0 -c "api-linter -I /common /out/*.proto --output-format summary"
-	docker run --user=$$(id -u):$$(id -g) --rm --entrypoint=sh -v "${PWD}/../common/v1":/common -v "${PWD}"/v1alpha1/:/out -w /out ghcr.io/docker-multiarch/google-api-linter:1.36.0 -c "api-linter -I /common /out/*.proto --output-format github --disable-rule=core::0192::has-comments --disable-rule=core::0136 --disable-rule=core::0127 --set-exit-status"
```
Comment (glimchb) [inline storage/Makefile:18]: tracking lint ignores #195

**Review by jainvipin — APPROVED** [review]
LGTM!

**Review by mkalderon — APPROVED** [review]
Looks good. thanks.

**Pull request merged**